<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конфигурация ДДИИ</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="title">
        <string>Настройка ВИП</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>SiPM</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_pwm_sipm">
                 <property name="text">
                  <string>17.806</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_hvip_sipm">
                 <property name="text">
                  <string>34</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>k pwm</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>V, В</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Cherenkov</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_hvip_ch">
                 <property name="text">
                  <string>28</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_pwm_ch">
                 <property name="text">
                  <string>20.34</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>k pwm</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>V, В</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>PIPS</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_pwm_pips">
                 <property name="text">
                  <string>17.4</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_hvip_pips">
                 <property name="text">
                  <string>40</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>V, В</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>k pwm</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="title">
        <string>Настройка МПП</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5" columnstretch="1,0,0">
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>Электроны</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_18">
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>7</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_15">
               <property name="leftMargin">
                <number>7</number>
               </property>
               <property name="topMargin">
                <number>7</number>
               </property>
               <property name="rightMargin">
                <number>7</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>7</number>
               </property>
               <item row="4" column="0">
                <widget class="QLabel" name="label_13">
                 <property name="text">
                  <string>3 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_3">
                 <property name="text">
                  <string>800</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>1.6 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_0_5">
                 <property name="text">
                  <string>160</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_0_8">
                 <property name="text">
                  <string>340</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_1_6">
                 <property name="text">
                  <string>680</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>0.5 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_0_1">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>40</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>0.1 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>0.8 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_5">
                 <property name="text">
                  <string>900</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>5 МэВ</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="title">
             <string>Протоны</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_17" rowstretch="0">
             <property name="leftMargin">
              <number>7</number>
             </property>
             <property name="topMargin">
              <number>7</number>
             </property>
             <property name="rightMargin">
              <number>7</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>7</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_16">
               <property name="leftMargin">
                <number>7</number>
               </property>
               <property name="topMargin">
                <number>7</number>
               </property>
               <property name="rightMargin">
                <number>7</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>7</number>
               </property>
               <item row="0" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_10">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>1000</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_60">
                 <property name="text">
                  <string>1210</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLineEdit" name="lineEdit_lvl_30">
                 <property name="text">
                  <string>1110</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_14">
                 <property name="text">
                  <string>10 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>60 МэВ</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_16">
                 <property name="text">
                  <string>30 МэВ</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_8">
            <property name="title">
             <string>Общее</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_14" rowstretch="0">
             <property name="leftMargin">
              <number>7</number>
             </property>
             <property name="topMargin">
              <number>7</number>
             </property>
             <property name="rightMargin">
              <number>7</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <property name="horizontalSpacing">
              <number>7</number>
             </property>
             <property name="verticalSpacing">
              <number>5</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_13">
               <property name="leftMargin">
                <number>7</number>
               </property>
               <property name="topMargin">
                <number>7</number>
               </property>
               <property name="rightMargin">
                <number>7</number>
               </property>
               <property name="bottomMargin">
                <number>5</number>
               </property>
               <property name="spacing">
                <number>7</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Интр. опроса, с</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLineEdit" name="lineEdit_cfg_mpp_id">
                 <property name="text">
                  <string>15</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="lineEdit_interval">
                 <property name="minimumSize">
                  <size>
                   <width>80</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>720</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_18">
                 <property name="text">
                  <string>МПП ID</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_save_mpp">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
